app/hotels/src/appRegistry/HotelsStandalonePackage.js

```diff
@@ -17,7 +19,18 @@ type Props = {|
   checkout?: string,
 |};
 
+type State = {
+  key: string,
+  isTransitioning: boolean,
+  index: number,
+  routes: mixed[],
+};
```

Comment:
Do we need to type state here? Probably `*` will do just fine? If not, maybe in GestureController. Right now, you have it defined twice.